uberfire-api/src/main/java/org/uberfire/workbench/type/Category.java

```diff
@@ -0,0 +1,10 @@
+package org.uberfire.workbench.type;
+
+public enum Category {
```

Comment:
We've previously tried to keep AppFormer clean of KIE related terms (IDK if this still happens!?!?) as AppFormer was meant to be a generic _workbench_ framework and not specific for KIE. Therefore I question whether we should have this ```enum``` in AppFormer. At best have some abstraction that is implemented in ```kie-wb-common```?

It is however a while since I was directly involved with AppFormer so I could be out of date.. hence happy to defer the decision to @ederign or @mbarkley

Reply:
Thanks for tagging me in. New plan is that it doesn't make sense to split AppFormer from the rest of our platform artificially when the reality is that we mostly care about KIE-related use cases.

So KIE terminology is fine and after we fix the build, AppFormer will depend on droolsjbpm-knowledge. AppFormer should not depend on drools, jbpm, or optaplanner directly though.